pathways/nl-NL.yml

```diff
@@ -1,140 +1,140 @@
----

```

Comment:
Line endings ... it was the only one set to `CRLF` not `LF` so I changed it. We could remove that check in the future.